### PR TITLE
Add a template workflow example for building mlir-air with wheels

### PR DESCRIPTION
**Pull request by erwei-xilinx**

### CODE REVIEW

File: utils/build-mlir-air-using-wheels.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+MLIR_AIE_WHL_DIR="mlir_aie_wheel"
+
+mkdir -p $BUILD_DIR
+mkdir -p $INSTALL_DIR
+cd $BUILD_DIR
+mkdir -p $MLIR_WHL_DIR
+mkdir -p $MLIR_AIE_WHL_DIR
+
+MLIR_DISTRO="mlir_no_rtti-18.0.0.2023121521+d36b483-py3-none-manylinux_2_27_x86_64.manylinux_2_28_x86_64.whl"
+MLIR_AIE_DISTRO="mlir_aie_no_rtti-0.0.1.2023121602+5631ba5c-py3-none-manylinux_2_28_x86_64.whl"
```
Comment (makslevental) [inline utils/build-mlir-air-using-wheels.sh:35]: you can just do 

```
export PIP_FIND_LINKS="https://github.com/Xilinx/mlir-aie/releases/expanded_assets/mlir-distro https://github.com/Xilinx/mlir-aie/releases/expanded_assets/latest-wheels"
pip download mlir_no_rtti==18.0.0.2023121521+d36b483
pip download mlir_aie_no_rtti==0.0.1.2023121602+5631ba5c
```

which will work on all platforms (not just linux - ie you don't need the `manylinux_2_28_x86_64` at the end).

File: utils/build-mlir-air-using-wheels.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    -DLLVM_ENABLE_ASSERTIONS=ON \
+    -DPython3_FIND_VIRTUALENV=ONLY \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DLLVM_TARGETS_TO_BUILD=X86 \
+    -DLLVM_HOST_TRIPLE=x86_64-unknown-linux-gnu \
+    -DLLVM_ENABLE_RTTI=OFF \
+    |& tee cmake.log
+
+ninja |& tee ninja.log
+ninja install |& tee ninja-install.log
```
Comment (makslevental) [inline utils/build-mlir-air-using-wheels.sh:81]: Can you please replace `|&` with `2>&1 |`; the former isn't portable.

**Review by makslevental — COMMENTED** [review]
LGTM (modulo suggestions)

File: utils/build-mlir-air-using-wheels.sh (hunk 1 above)
Comment (erwei-xilinx) [reply utils/build-mlir-air-using-wheels.sh]: Thanks for the pointer! Script is updated.

File: utils/build-mlir-air-using-wheels.sh (hunk 2 above)
Comment (erwei-xilinx) [reply utils/build-mlir-air-using-wheels.sh]: Thanks! Updated.

**Comment by newling** [comment]
@makslevental @erwei-xilinx any chance we can get CI to use wheels, too? 

Semi-related: is it just me, or is CI slower in 2024? https://github.com/Xilinx/mlir-air/pull/371/checks took 2h28 mins.

**Pull request merged**